<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorderCfger</class>
 <widget class="QGroupBox" name="BorderCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>230</height>
   </rect>
  </property>
  <property name="title">
   <string>Border</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="checked">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelStyle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStyle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>8</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpeed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDirection">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpeed">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
